comments/stakeholder_annotations.md

```diff
@@ -96,7 +92,28 @@ The painpoint annotation data goes into the UserMapAnnotationBag (a database tab
 
 ### Output data
 
-An API is called which aggregates all of the UserMapAnnotationBags for the target map together, and returns a tabular data file similar to the one outlined in the objective. 
+An API is called which aggregates all of the UserMapAnnotationBags for the target map together. API calls are formatted as follows:
+
+    https://staging.ecosystem.guide/api/annotations/report?map=<guest_map_path>
+
+where `guest_map_path` is the guest map path property for the target map (can be viewed in the Admin panel).
+
+A user must be logged in with an SMP user account to use this API.
+
+Making a request to this API (or hitting it in a browser) will return a .csv formatted as follows:
+
+| Commented By (email) | Comment | Category | Stakeholder/Painpoint | Stakeholder Name | Painpoint Name |
+|----------------------|---------|----------|-----------------------|------------------|----------------|
+| joe.bloggs@example.com | Some comment text | Suprising | Painpoint | A stakeholder | A painpoint |
```

Comment:
From testing with some actual maps content, there will be things commented on that are neither stakeholders nor painpoints. For now having these columns blank (and adding the iri of the subject) is probably fine.

Followup would be a more adaptive export format that instead of having fixed columns produced some kind of tidy output where ontology classes could map to new columns.

Reply:
Returning subject IRI is a good idea, but unfortunately not what the API is currently doing. I'll merge this documentation update in as-is, and then do a quick update to the report code today to return IRIs when I have a bit of time.